src/chains/filecoin/filecoin/src/api.ts

```diff
@@ -99,8 +186,8 @@ export default class FilecoinApi implements types.Api {
     return {};
   }
 
-  async "Filecoin.GanacheMineTipset"(): Promise<SerializedTipset> {
-    await this[_blockchain].mineTipset();
-    return this[_blockchain].latestTipset().serialize();
+  async "Ganache.MineTipset"(): Promise<SerializedTipset> {
```

Comment:
I will make my one remark, but you can tell me whatever you want it to be; I'm not going to spend more time bikeshedding about this.

First, using the `Filecoin.` prefix here for ganache specific stuff is misleading. It makes it seem like it's part of the Filecoin schema, but it's not. The schema doesn't even have those prefixes; it's clients/nodes that adhere to the schema that actually add/remove the prefix. You can see the schema [here](https://github.com/filecoin-shipyard/js-lotus-client-schema) or the actual methods [here](https://github.com/filecoin-shipyard/js-lotus-client-schema/blob/master/mainnet/index.js) doesn't actually refer to the `Filecoin.` prefix; it's part of the API specification rather than schema specification. This throws out the idea that we should put our custom messages in the same namespace as the official schema.

Then there's what ganache already does. Ethereum ganache uses `evm_mine` for the equivalent here. I think the `evm_` prefix here is misleading to begin with. It makes it seem like this is a standardized method, but it's actually only implemented anywhere with Ganache. Further, we aren't implementing the VM here, so we can't do something `FVM.` without being misleading. So that throws out the window of "what is existing ganache behavior".

Since I believe it's important to be consistent at the local areas rather than global (it's more important to be consistent with the rest of the Filecoin api than it is to be consistent with the rest of the Ganache codebase) because thats what developers are familiar with, `Ganache.` felt like very apt namespace here. Using `Truffle` or `TruffleSuite` would both either be confusing or nonintuitive. I also think it's important that there *is* a namespace rather than no namespace, because the Filecoin API clearly wants to create namespaces.

With that all said, I'm happy for yall to tell me whatever you want, but that's the reasoning here.

Reply:
I've always thought of `evm_` as our way of saying "this is a custom namespace, but we invite other tools to copy us". Whereas had we done `ganache_` it would encourage a divide in the standardization of the tools.

@tcoulter, any thoughts?

Reply:
Couple thoughts: 

- All good points @seesemichaelj.
- So far, I have no strong preferences. But do have some thoughts, below. 
- I'm open to a prefix that's less branded, per @davidmurdoch's comment. @davidmurdoch, do you have a suggestion for what this could be? Maybe `Miner.MineTipset`? I forget though if the `Miner` namespace is already used. Other prefixes: `Debug`. `Test`. `Develop`. `Dev`? 
- Separately, does using a namespace other than `Filecoin` make the method harder to call from convenience libraries like `lotus-client-rpc`? If so, I'd lean toward keeping the `Filecoin` prefix.

Reply:
You cannot execute this method from `lotus-client-rpc` because that library only uses methods from the schema. Either filecoin would need to add this to the official schema somewhere (I'm doubtful since this method makes no sense outside of ganache) or it would need to be added separately. `filecoin.js` would need to add it as well regardless and doesnt make a difference to them

FWIW, `evm_` didn't get picked up by the community and `evm_mine` is still, several years later, exclusively a Ganache method. I honestly don't see it any different here, and as a dev I'd rather know that this is specific to ganache

Reply:
A comment @tcoulter made on another piece of code made me realize that we can have this method work with the current `lotus-client-rpc` if and only if it has the `Filecoin.` prefix. This is due to `lotus-client-rpc` [hardcoding](https://github.com/filecoin-shipyard/js-lotus-client-rpc/blob/master/index.js#L38) the `Filecoin.` prefix

However, I have every belief that `lotus-client-rpc` was designed around the fact that there is currently only one namespace. A very simple PR over there would enable `Ganache.MineTipset` to be callable from `provider.ganacheMineTipset()` or `provider.mineTipset()` depending on Protocol Labs' preference.

Reply:
Good points. 

Unless you have strong feelings @davidmurdoch I'm happy to lave it `Ganache.<method>`.

Reply:
Thanks Tim. Ship it!